ecommerce/models.py

```diff
@@ -80,6 +80,9 @@ class Position(models.Model):
     name = models.CharField(max_length=255)
     quantity = models.PositiveSmallIntegerField(default=0)
     price = models.FloatField(blank=True, null=True)
+    code = models.CharField(max_length=255, default='')
```

Comment:
SE doesn't have such fields. We should cast this to abstract model, to achieve  extensibility at a client side. Let's create todo for it

Reply:
@artemiy312 , Every line of code i added in this PR is a bad arch decision. Including this fields of course. But it's because awful view-cart-order-position arch in a whole. 
So, we can create a task to redesign this mech in a whole. Incremental improvements on this arch will be too painful.

I haven't created the task because of some strategical thoughts. Let's discuss them in direct, then we'll create the task with our dicsussion summary and decisions